src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import './styles/index.scss';
+import Navbar from './components/Navbar.jsx';
+import Cover from './components/Cover.jsx';
+
+function App() {
+  return (
+    <div>
+      <Cover />
+      <Navbar />
+    </div>
```

Comment:
Podríamos utilizar fragments para evitar un div aquí. Además, ¿por qué `Navbar` viene después si supuestamente es un header?
```suggestion
    <>
      <Navbar />
      <Cover />
    </>
```